web-components/src/components/organisms/CancelButtonFooter/CancelButtonFooter.stories.tsx

```diff
@@ -0,0 +1,20 @@
+import { ComponentMeta, ComponentStory } from '@storybook/react';
+
+import { CancelButtonFooter } from './CancelButtonFooter';
+
+export default {
+  title: 'organisms/CancelButtonFooter',
+  component: CancelButtonFooter,
+} as ComponentMeta<typeof CancelButtonFooter>;
+
+const Template: ComponentStory<typeof CancelButtonFooter> = args => (
+  <CancelButtonFooter {...args} />
+);
+
+export const Default = Template.bind({});
+
+Default.args = {
+  onCancel: () => alert('cancel'),
```

Comment:
We could use `action` from Storybook here instead of using `alert`: https://storybook.js.org/docs/react/essentials/actions